services/core/BACnetProxy/bacnet_proxy/agent.py

```diff
@@ -390,15 +381,16 @@ def indication(self, apdu):
 
     # Handler for ConfirmedCOVNotificationRequests, forwards the notification to the appropriate driver agent
     def do_ConfirmedCOVNotifcationRequest(self, apdu):
-        if isinstance(apdu, ConfirmedCOVNotificationRequest):
-            for sub in subCovContexts.itervalues():
-                if apdu.monitoredObjectIdentifier == sub.monitoredObjectIdentifier and apdu.pduSource == sub.address:
-                    point_name = sub.point_name
-                    break
-            if point_name:
-                self.forward_cov_callback(apdu.pduDestination, point_name, apdu.listOfValues)
-            else:
-                _log.debug("Device {} does not have a subscription context.".format(apdu.monitoredObjectIdentifier))
+        sys.stdout.write("we've got a notification")
```

Comment:
Is there a reason we are using stdout instead of _log.debug?

Reply:
Woops. This was removed but didn't make it into the pull request.